manim/mobject/coordinate_systems.py

```diff
@@ -59,6 +59,47 @@
 class CoordinateSystem:
     """
     Abstract class for Axes and NumberPlane
+
+    Examples
+    --------
+
+    .. manim:: CoordSysExample
+        :save_last_frame:
+
+        class CoordSysExample(Scene):
+            def construct(self):
+                # When drawing Axes, the location of the ticks
+                # on the axes depend on x_range and y_range.
+                grid = Axes(
+                    x_range=[0, 1, 0.05],  # The number of decimal places is determined by the step size.
```

Comment:
`#The number of decimal places is determined by the step size.` -> `# step size determines num_decimal_places.`